python/cugraph/cugraph/tests/traversal/test_paths.py

```diff
@@ -37,13 +38,29 @@
 DISCONNECTED_GRAPH = CONNECTED_GRAPH + "8,9,4"
 
 
+paths_test_data = {
+    "shortest_path_length_1_1": 0,
+    "shortest_path_length_1_5": 2.0,
+    "shortest_path_length_1_3": 2.0,
+    "shortest_path_length_1_6": 2.0,
+    "shortest_path_length_-1_1": ValueError,
+    "shortest_path_length_1_10": ValueError,
+    "shortest_path_length_0_42": ValueError,
+    "shortest_path_length_1_8": 3.4028235e38,
+}
```

Comment:
Is this a placeholder for calls to `get_resultset()`, or are you thinking this is permanent due to the resutls being just single values?  If it's a placeholder, now is probably the time to replace it with `get_resultset()`, but either way a comment would be helpful.

Reply:
It's not a placeholder, but it's been renamed to `paths_golden_results`. I added a comment to explain why it's not a placeholder/added as a Resultset